src/eventstream-proxy/eventstream-proxy.base.ts

```diff
@@ -66,55 +68,86 @@ export abstract class EventStreamProxyBase extends WebSocketEventsBase {
 
   handleConnection(client: WebSocketEx) {
     super.handleConnection(client);
-    if (this.server.clients.size === 1) {
-      this.logger.log(`Initializing event stream proxy`);
-      Promise.all(this.connectListeners.map(l => l.onConnect()))
-        .then(() => {
-          this.setCurrentClient(client);
-          this.startListening();
-        })
-        .catch(err => {
-          this.logger.error(`Error initializing event stream proxy: ${err}`);
-        });
-    }
+    client.on('message', async (message: string) => {
```

Comment:
So we're now using a combination of Nest websocket message handlers (ie with `@SubscribeMessage('ack')`) and raw websocket message handlers (with `on('message')`). Maybe should choose one or the other?

Reply:
Maybe. I'm not sure how nest works 😆